src/classes/os-utils.ts

```diff
@@ -33,7 +33,7 @@ export default class OsUtils {
    */
   public static getOS = async () : Promise<string | null> => {
     try {
-      const os : UserConfigJson = await OsUtils.readUserConfig();
+      const os : UserConfigJson = await userConfigUtils.readUserConfig();
```

Comment:
Class names should be captitalized
```suggestion
      const os : UserConfigJson = await UserConfigUtils.readUserConfig();
```